lib/auth/webauthncli/fido2.go

```diff
@@ -526,16 +526,28 @@ func runOnFIDO2Devices(
 	prompt runPrompt, passwordless bool,
 	filter deviceFilterFunc,
 	deviceCallback deviceCallbackFunc) error {
-	devices, err := findSuitableDevicesOrTimeout(ctx, filter)
+	// Do we have readily available devices?
+	prompted := false
+	devices, err := findSuitableDevices(filter, make(map[string]struct{}))
```

Comment:
I looked up this map param, looks like it is always passed as just an empty map, does it need to be a parameter?

Reply:
Here it doesn't make much of a difference, but it usually outlives individual findSuitableDevices calls. I'll make everything share the same map.